main.go

```diff
@@ -26,8 +26,8 @@ func main() {
 		token      = fs.String("token", "", "Fastly API token (required)")
 		serviceIDs = stringslice{}
 		addr       = fs.String("endpoint", "http://127.0.0.1:8080/metrics", "Prometheus /metrics endpoint")
-		namespace  = fs.String("namespace", "", "Prometheus namespace (optional)")
-		subsystem  = fs.String("subsystem", "", "Prometheus subsystem (optional)")
+		namespace  = fs.String("namespace", "fastly", "Prometheus namespace (optional)")
+		subsystem  = fs.String("subsystem", "rt", "Prometheus subsystem (optional)")
```

Comment:
-1, please leave them blank.

Reply:
So, this one is the meat of it.  I'm a little unclear what functionality we're looking to offer by not providing a default namespace.

even the core prometheus bindings provide fixed namespaces

go_
process_
promhttp_

I would also reference https://prometheus.io/docs/instrumenting/writing_exporters/#naming
```
Metric names for applications should generally be prefixed by the exporter name, e.g. haproxy_up.
```
Could you possibly clarify what use case you are looking to cover?

I'm pushing on this one as ideally I'd like to build up to making canned grafana dashboards available to folks, but that becomes problematic if there is not a standard for the metric names (which includes a standard and default namespace and subsystem)